app/views/shared/_newsletter.html.haml

```diff
@@ -0,0 +1,35 @@
+.row
+  .medium-10.medium-offset-1.columns
+    %h2 Subscribe to our newsletter
+    %p Summary of what's in the newsletter. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+
+    -# Mailchimp Signup Form
+    #mc_embed_signup
```

Comment:
Can we use a surrounding container class to restrict the size of the input field? e.g. `medium-4` ?

Reply:
@despo I tried this but I didn't really like how it looked. Let me do it again and paste a screenshot, we can decide from there.

Reply:
@despo how about this? Also, I just realized we will have to convert this section to bootstrap as part of the migration, we can revisit the design then if we are not satisfied with how this section looks (which I am not). For now I just want to put something out there so people can subscribe.

<img width="1440" alt="Screenshot 2021-04-03 at 10 41 09 pm" src="https://user-images.githubusercontent.com/5873816/113499800-ba50dc00-94cd-11eb-9662-09af2194ae6c.png">